### ISSUE TITLE
Using pgAdmin is very cumbersome (RM #6321)

### ISSUE THREAD

**Issue by dpage** [pgadmin-org/pgadmin4#4223]
***Issue migrated from Redmine: https://redmine.postgresql.org/issues/6321***
*Originally created by **Matej Spiller Muys** at 2021-03-17 07:04:16 UTC.*

I love pgAdmin now that it behaves as an app. UI is fast, but it is very hard to use.

For example: Dashboard too complex view. To view table data one one table I need to click 9 times.

- click on connection
- click on databases
- click on database
- click on schemas
- click on public
- click on Tables
- click on table
- click on grid
- it loads the full table (couple of minutes)

Suggestions:
- simple dashboard vs complex dashboard (toggle?) -> one simple togggle to filter out every thing ... I should see databases with tables & views only ... 99.9% of time I don't care about collation, domains, fts, functions, foreign tables, etc.
- view data should display just data occupying most of the space (remove query editor)
- view data should load only first 1000 rows (now it loads terabytes of data with STOP button disabled ... yikes)
- add ability to STOP loading data for view data (now it is disabled)
- when opening query editor I should not see (Dashboard, Properties, SQL, Statistics, Dependencies, Dependents) ... they are properties of a table not a query
- when entering ssh password I don't have option save password enabled

**Comment by imcdona** [comment]
tldr; Upon logging into pgAdmin it should easy to examine a database, it's tables, and to insert and select data. Currently this is not the case.

Original request:
> simple dashboard vs complex dashboard (toggle?) -> one simple togggle to filter out every thing ... I should see databases with tables & views only ... 99.9% of time I don't care about collation, domains, fts, functions, foreign tables, etc.

Suggested workaround:
> You can turn off object types you don't care about under File -> Preferences -> Browser -> Nodes.

Maybe a toggle in preferences to enable the additional objects rather than the other way around. I don't know for certain but I imagine a large percentage of users are accessing pgAdmin to examine tables manipulate data. Making it easy to examine tables and run SQL without having to first understand the underlying hierarchy of objects would make things much easier.

There is the administration aspect of the Postgres and then there is the database management aspect. The focus seems to be on server administration which, while valuable, I don't feel is the "primary use case" for the app. I could be wrong though.

**Comment by adityatoshniwal** [comment]
I can confirm the only thing to fix here is fixed (`add ability to STOP loading data for view data (now it is disabled)`).
Stop button can be used to stop loading data.
